Source/Plugins/Core/com.equella.core/js/tsrc/search/SearchPage.tsx

```diff
@@ -16,55 +16,88 @@
  * limitations under the License.
  */
 import * as React from "react";
-import { useEffect, useState } from "react";
+import { useEffect, useRef, useState } from "react";
 import {
   templateDefaults,
   templateError,
   TemplateUpdateProps,
 } from "../mainui/Template";
 import { languageStrings } from "../util/langstrings";
 import {
+  Card,
+  CardActions,
+  CardContent,
+  Grid,
   IconButton,
   List,
-  ListItem,
   ListSubheader,
+  TablePagination,
   TextField,
-  Grid,
-  Card,
-  CardContent,
 } from "@material-ui/core";
 import SearchIcon from "@material-ui/icons/Search";
-import { searchItems } from "./SearchModule";
+import { defaultPagedSearchResult, searchItems } from "./SearchModule";
 import * as OEQ from "@openequella/rest-api-client";
 import SearchResult from "./components/SearchResult";
 import { generateFromError } from "../api/errors";
 import {
   getSearchSettingsFromServer,
   SearchSettings,
+  SortOrder,
 } from "../settings/Search/SearchSettingsModule";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  cardAction: {
+    display: "flex",
+    justifyContent: "center",
+  },
+});
 
 const SearchPage = ({ updateTemplate }: TemplateUpdateProps) => {
+  const classes = useStyles();
   const searchStrings = languageStrings.searchpage;
-  const [searchResultItems, setSearchResultItems] = useState<
-    OEQ.Search.SearchResultItem[]
-  >([]);
+
+  const [rowsPerPage, setRowsPerPage] = useState<number>(10);
+  const [currentPage, setCurrentPage] = useState<number>(0);
+  const [sortOrder, setSortOrder] = useState<SortOrder>(SortOrder.RANK);
+  const [pagedSearchResult, setPagedSearchResult] = useState<
+    OEQ.Common.PagedResult<OEQ.Search.SearchResultItem>
+  >(defaultPagedSearchResult);
 
   /**
-   * Update the page title and do a default search.
+   * Construct a standard search criteria.
+   */
+  const standardParams: OEQ.Search.SearchParams = {
```

Comment:
Are there any non-standard params? Maybe I'm missing something, but it seems like this could just be:

```suggestion
  const searchParams: OEQ.Search.SearchParams = {
```